data/people/EleniIlkou.ttl

```diff
@@ -5,8 +5,8 @@ kg:EleniIlkou a schema:Person ;
   schema:name "Eleni Ilkou" ;
   schema:role kg:ESR ;
   schema:photo "EleniIlkou.png" ;
-  schema:namePrefix "" ;
-  schema:email "" ;
+  schema:namePrefix "Eleni Ilkou" ;
```

Comment:
@naiayti the name prefix is reserved for titles like "Prof. Dr.". The web page would name you now "Eleni Ilkou Eleni Ilkou" everywhere. I don't think that you want to have this effect :wink: